rules/source-control-tools/rule.md

```diff
@@ -0,0 +1,143 @@
+4. **Reducing Risks:** it can help reduce risks associated with software development. They can assist in identifying and fixing bugs more quickly, and they can also help protect projects from data loss.
+5. **Improving Code Quality:** it can help improve code quality by encouraging developers to write cleaner and more maintainable code.
+6. **Promoting Communication:** it can help promote communication among developers. They provide a way for developers to share their changes and collaborate on projects.
+
+### Disadvantages of Version Control Software
+
+1. **Complexity:** Version control software can be complex to learn and use. This can be a barrier for beginner developers.
+2. **Cost:** Commercial version control software can be expensive. However, many free open-source solutions are available.
+3. **Performance:** Version control software can negatively impact the performance of the systems they are used on. This is particularly true for large projects.
+4. **Security:** Version control software can be vulnerable to attacks. It is important to take security measures to protect the data stored in a version control repository.
+
+::: greybox
+**🧐 Did You Know?**  
+One of the earliest version control systems, named **SCCS (Source Code Control System)**, was developed by Marc J. Rochkind at Bell Labs in 1972. At that time, the need to efficiently manage changes to source code was just beginning to emerge in the software industry.  
+This system enabled the tracking of revisions for individual files, a significant innovation for its time. It's from these early initiatives that modern version control tools like Git and Mercurial have evolved, offering far more advanced features and collaborative code management capabilities.
+:::
+
+### [GitKraken](https://www.gitkraken.com/)
+
+GitKraken is a renowned suite of developer tools designed to enhance productivity in software development. It has been evolving for nearly a decade, reflecting a continuous commitment to improving the efficiency and effectiveness of developers and their teams. GitKraken began with desktop-centric and IDE-based tools, like the GitKraken Client and GitLens for Visual Studio Code, both of which have become integral to the workflows of tens of millions of developers worldwide​​.
+
+The company behind GitKraken, Axosoft, has been recognized for its significant contributions to the developer community. For instance, Eric Amodio, the Chief Technology Officer at GitKraken, was awarded the Microsoft MVP Award for Developer Technologies in 2023. This award highlights his exceptional contributions to the developer community, particularly through GitLens, a popular Git extension for Visual Studio Code created by Amodio in 2016. GitLens has become a crucial tool in the daily coding processes of millions of developers, with its installation numbers surpassing 23 million.
+
+#### Pros:
```

Comment:
@PothieuG 

- [ ] 1. Please add a green tick here ✅

Same for the others Pros